### PR TITLE
Fix/overlaps

### PR DESCRIPTION
**Pull request by rmit-s3722599-Minh-Le**
Buttons and text overlapping. #184

-Fixed text and Buttons overlapping
-Buttons written in PseudoBox

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -183,19 +205,38 @@ class IndexPage extends Component {
...
                     </div>
                   </div>
-                  <div className='db center w-auto pv4 pb0-ns ph4 tc pointer'>
-                    <a className='btn-red no-underline pa3 db br-pill tf-lato b v-mid w-80 m-auto'
-                      href='mailto:joelwass@theteacherfund.com?subject=Application'>
-                       Apply Today
-                    </a>
-                  </div>
+                  <Box w='100%' p={4} display='block' marginLeft='0' marginRight='0' marginTop='1px'>
+                    {/* Appky Today Button */}
```
Comment (joelwass) [inline pages/index.js:216]: nit: typo

**Review by joelwass — APPROVED** [review]

**Comment by joelwass** [comment]
Thanks so much for this PR! It all looks amazing!

🥳 🎊

**Comment by joelwass** [comment]
Fixes #184

File: pages/index.js (hunk 1 above)
Comment (rmit-s3722599-Minh-Le) [reply pages/index.js:216]: Sorry about that!!

**Pull request merged**